### ISSUE TITLE
CanvasItem methods in Godot 4 lacking some parameters that were present in 3.x

### ISSUE THREAD

**Issue by AndersonDeMatos** [godotengine/godot#59683]
### Godot version

v4.0.alpha5.official [d7d528c15]

### System information

Linux Pop!_OS 21.10, Nvidia GT710, Nvida X Server Settings 470.75.01

### Issue description

I'm porting a project from Godot 3.4.4 to Godot 4, and when I needed to write the method draw_polygon() it complained about having too much parameters. I needed to remove the "normal_map" and "antialiased" parameters to make it work.
Other draw methods lack the those parameters too.
For my project it is ideal to have the anti-alias enabled to make it look better.

### Steps to reproduce

Basically just try to call draw_polygon() with the same amount of parameters as you would do in version 3.x

**Comment by Calinou** [comment]
Antialiased line drawing was restored in https://github.com/godotengine/godot/pull/43828, but this wasn't done for polygon drawing yet.

In the meantime, you could try porting https://github.com/godot-extended-libraries/godot-antialiased-line2d to 4.0 (it's a 3.x add-on).

**Comment by clayjohn** [comment]
Normal maps are now specified as part of the CanvasTexture rather than specifying them on the Canvasitem itself

**Comment by Mickeon** [comment]
I was going to give this a go but this does not feel particularly desirable? Plus, it's not particularly straightforward.
